vite.config.ts

```diff
@@ -45,7 +45,9 @@ const getDefine = async ({ mode }: UserConfig) => {
     path: VAULT_PATH
   });
 
-  return Object.entries(secrets).reduce((acc, [key, value]) => {
+  //Ovverride VITE_HOME_LAMBDAS_BASE_URL with .env value if it exists
+  const merged = {...secrets,VITE_HOME_LAMBDAS_BASE_URL: env.VITE_HOME_LAMBDAS_BASE_URL ?? secrets.VITE_HOME_LAMBDAS_BASE_URL};
+  return Object.entries(merged).reduce((acc, [key, value]) => {
```

Comment:
I'm guessing this is something you have been using locally that shouldn't have been committed?
This PR can be approved as soon as this is removed and the other comments are addressed